### PR TITLE
feat(statics): Add ERC20 tokens to OFC support

### PR DESCRIPTION
**Pull request by amit-yeles-bitgo**
Ticket: BG-58420

## Description

Added OFC support to ERC20 tokens (ELF ENS EURS FUN FXS)

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -27,14 +26,15 @@ import {
   tsolToken,
   tstellarToken,
   ttronToken,
+  AccountCoin,
```
Comment (stevenharrisBG) [inline modules/statics/src/coins.ts:29]: This looks correct. Alpha sort should be lower case first for readability.

**Review by stevenharrisBG — CHANGES_REQUESTED** [review]
there are errors, here.  Requesting change to block, will update momentarily.

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -1041,7 +1041,12 @@ export const coins = CoinMap.fromCoins([
   ofcerc20('ofcdep', 'Deap Coin', 18, UnderlyingAsset.DEP),
   ofcerc20('ofcdfi', 'DeFiChain', 8, UnderlyingAsset.DFI),
   ofcerc20('ofcdydx', 'dYdX', 18, UnderlyingAsset.DYDX),
+  ofcerc20('ofcelf', 'Aelf', 18, UnderlyingAsset.ELF),
+  ofcerc20('ofcens', 'Ethereum Name Service', 18, UnderlyingAsset.ENS),
+  ofcerc20('ofceurs', 'Stasis EURS', 18, UnderlyingAsset.EURS),
   ofcerc20('ofcftt', 'FTX Token', 18, UnderlyingAsset.FTT),
+  ofcerc20('ofcfun', 'FunFair', 18, UnderlyingAsset.FUN),
```
Comment (stevenharrisBG) [inline modules/statics/src/coins.ts:1048]:
```suggestion
  ofcerc20('ofcfun', 'FunFair', 8, UnderlyingAsset.FUN),
  
```

File: modules/statics/src/coins.ts [hunk 3]
```diff
@@ -1041,7 +1041,12 @@ export const coins = CoinMap.fromCoins([
   ofcerc20('ofcdep', 'Deap Coin', 18, UnderlyingAsset.DEP),
   ofcerc20('ofcdfi', 'DeFiChain', 8, UnderlyingAsset.DFI),
   ofcerc20('ofcdydx', 'dYdX', 18, UnderlyingAsset.DYDX),
+  ofcerc20('ofcelf', 'Aelf', 18, UnderlyingAsset.ELF),
+  ofcerc20('ofcens', 'Ethereum Name Service', 18, UnderlyingAsset.ENS),
+  ofcerc20('ofceurs', 'Stasis EURS', 18, UnderlyingAsset.EURS),
```
Comment (stevenharrisBG) [inline modules/statics/src/coins.ts:1046]:
```suggestion
  ofcerc20('ofceurs', 'Stasis EURS', 2, UnderlyingAsset.EURS),
```

**Review by stevenharrisBG — CHANGES_REQUESTED** [review]
Decimal places should match the UnderlyingAsset, I'm sure.

**Review by byhow — APPROVED** [review]

**Pull request merged**